MAINTAINERS.md

```diff
@@ -0,0 +1,15 @@
+This page lists all active maintainers of this repository. If you were a
+maintainer and would like to add your name to the Emeritus list, please send us a
+PR.
+
+See [CONTRIBUTING.md](https://github.com/grpc/grpc-community/blob/master/CONTRIBUTING.md)
+for general contribution guidelines.
+
+## Maintainers (in alphabetical order)
+- [mit-mit](https://github.com/mit-mit), Google Inc.
+- [nichite](https://github.com/nichite), Google Inc.
+- [szakarias](https://github.com/szakarias), Google Inc.
+- [sigurdm](https://github.com/sigurdm), Google Inc.
+
+## Emeritus Maintainers (in alphabetical order)
+- [jakobr-google](https://github.com/jakobr-google), Google Inc.
```

Comment:
Do we need an emeritus list?
Earlier maintainers and contributors could belong in the AUTHORS file...

Reply:
That's what the open source team asked for. Apparently this is a common CNCF template